.github/workflows/test-deploy-wf-service.yml

```
@@ -176,21 +190,21 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}-ee
           tags: |
             type=raw,value=${{ inputs.environment }}
-            type=raw,value=${{ github.event_name == 'workflow_dispatch' && steps.lastcommit.outputs.shorthash || inputs.sha }}-${{ inputs.environment }}
+            type=raw,value=${{ env.SHORT_HASH }}-${{ github.env.SUBMODULE_SHORT_HASH }}-${{ inputs.environment }}
             type=raw,value=latest,enable=${{ inputs.environment == 'prod' }}
             type=sha,format=short
             
```

yamllint

[error] trailing spaces

(trailing-spaces)